docs/howto/openstack/neutron/create-security-groups.md

```diff
@@ -0,0 +1,59 @@
+# How to Create Security Groups
+
+By definition, security groups are "[...] _sets of IP filter rules that are applied to all project instances, which define networking access to the instance. Group rules are project specific; project members can edit the default rules for their group and add new rule sets_". [[1](https://docs.openstack.org/nova/latest/admin/security-groups.html)]
+
+To create a security group click on "_Security Groups_" in the left-hand navigation menu:
+
+![navigation-panel](assets/create-security-groups/01-navigation-panel.png)
+
+and then click on "_Create new Security Group_" in the top-right corner:
+
+![create-button](assets/create-security-groups/02-create-button.png)
+
+> An alternative way to create a Security Group is by clicking on "_Create ...._" button in the top bar.
+
+Now give the security group a name and description and choose in which region to create it, then click "_create_":
+
+![create-panel](assets/create-security-groups/03-name-creation.png)
+
+## Remove default ingress rules
+
+By default, a security group named "_default_" has been already created for you, allowing any sort of traffic from any source (ingress).  
+Click on it and select the "_Rules_" tab to view its rules:
+
+![default-rules](assets/create-security-groups/04-default-rules.png)
+
+This is not recommended and a good approach is to either create and use a new security group than the default one **-or-** restrict ingress traffic to specific ports and sources.
```

Comment:
* Replace "This is not recommended and and a good approach is to" with "We recommend to".
* There needs to be a comma after "than the default one".
* Why are there hyphens around "or"?

Reply:
The hypens are from the old guide, and now they're gone.